.github/workflows/ci-{{app_name}}-infra-service.yml.jinja

```diff
@@ -31,27 +31,42 @@ on:
   #     - infra/{{ app_name }}/service/**
   #     - infra/modules/**
   #     - infra/test/**
-  #     - .github/workflows/ci-infra-service.yml
+  #     - .github/workflows/ci-{{ app_name }}-infra-service.yml
   # pull_request:
   #   paths:
   #     - infra/{{ app_name }}/service/**
   #     - infra/modules/**
   #     - infra/test/**
-  #     - .github/workflows/ci-infra-service.yml
+  #     - .github/workflows/ci-{{ app_name }}-infra-service.yml
   {% endif %}
   workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        default: "main"
+        description: Tag or branch or SHA to test
 
 jobs:
+  build-and-publish:
+    name: Build
+    uses: ./.github/workflows/build-and-publish.yml
+    with:
+      app_name: {{ app_name }}
+      ref: ${{ inputs.version || github.ref }}
```

Comment:
Oh, you'll have to escape the angle brackets you want to retain/the GH variables that are not Jinja variables.

Reply:
Ah, thanks didn't know that. Will do